src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/SqlInternalConnectionX.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if NET8_0_OR_GREATER
+
+using System;
+using System.Data;
+using System.Data.Common;
+
+namespace Microsoft.Data.SqlClientX
+{
+    internal class SqlInternalConnectionX : DbConnection
+    {
+        public SqlInternalConnectionX() { }
```

Comment:
This declaration is redundant, the compiler generates a default constructor if there are no constructor declarations.